jupyter_events/yaml.py

```diff
@@ -20,9 +20,10 @@ def dumps(stream):
 
 
 def load(fpath):
-    data = pathlib.Path(str(fpath)).read_text()
+    # coerce PurePath into Path, then read its contents
+    data = Path(str(fpath)).read_text()
     return loads(data)
```

Comment:
This out of scope for this PR, but these functions have some pretty bad names since they do different functionality based on the presence of the `s` character. It's worthwhile to change these to `load_stream()` and `load_path()`.

Reply:
`load/loads` are patterns that are consistent with the `json` and `yaml` libraries, so it makes sense for this wrapper to share the same convention.

Reply:
I see. Let's preserve parity then.